### PR TITLE
StyleLint plugin should work without configuration and just use default one

### PR DESCRIPTION
**Pull request by artyomtrityak**
Currently it throws exception for this config:

```js
plugins: [
  new StyleLintPlugin()
]
```

This is fix for this use case

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -90,4 +90,19 @@ describe('stylelint-webpack-plugin', function () {
...
+      context: './test/fixtures/test9',
+      entry: './index',
+      plugins: [
+        new StyleLintPlugin()
+      ]
+    };
+
+    return pack(assign({}, baseConfig, config))
+      .then(function (stats) {
+        expect(stats.compilation.errors).to.have.length(0);
```
Comment (JaKXz) [inline test/index.js]: Could you add the assertion for the `stats.compilation.warnings` as well (since the test title implies it)?

Comment (artyomtrityak) [inline test/index.js]: sure

Comment (artyomtrityak) [inline test/index.js]: I don't think we should requiring config, default one works great. Same as eslint

**Comment by artyomtrityak** [comment]
I don't think we should requiring config, default one works great. Same as eslint

**Comment by JaKXz** [comment]
@vieron I've had some time to think about this and I think it's a good idea. Do you have any objections? If not I'll release this as the next version :)

**Pull request merged**